project-management/src/main/java/life/qbic/projectmanagement/domain/model/measurement/NGSIndex.java

```diff
@@ -0,0 +1,19 @@
+package life.qbic.projectmanagement.domain.model.measurement;
+
+/**
+ * <b><record short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
```

Comment:
What is an `i5` or `i7` index? Providing some description here might help developers in the future if they are not aware of it.

Reply:
Good point 👍